Some small Bug fixes

stefanheimes opened the pull request:
- It is now possible to get the data even if the attribute isn't in the sort order list
- The static data "cache" use the id of the attribute.

stefanheimes committed:
Some small Bug fixes

- It is now possible to get the data even if the attribute isn't in the sort order list
- The static data "cache" use the id of the attribute.

stefanheimes committed:
Bugfix for caching.

stefanheimes committed:
Bugfix for the cache again.

discordier commented:
Please fix the code style violations:
https://travis-ci.org/github/MetaModels/attribute_geodistance/jobs/696158881#L584

stefanheimes committed:
Make travis happy

stefanheimes committed:
Add a field definition for contao.

- Add a simple field definition for the dca
- This should fix the problem mention in #12
